scripts/YaML.md

```diff
@@ -468,6 +470,8 @@ class ur_name_flags_v(IntEnum):
       - `out` is used for members that are write-only; if the member is a pointer, then the memory being pointed to is also write-only
       - `in,out` is used for members that are both read and write; typically this is used for pointers to other data structures that contain both read and write members
       - `nocheck` is used to specify that no additional validation checks will be generated.
+    + `desc` must also include the following annotation when describing a union: {`"tagged_by(param)"`}
+      - `tagged_by` is use to specify which parameter will be used as the tag for accessing the union.
```

Comment:
used*

Reply:
Can you clarify what you mean by this?

Reply:
I meant that it should be:

" `tagged_by` is used to specify" instead of "`tagged_by` is use to specify."

Reply:
👍